src/components/chart/statistics-chart.ts

```diff
@@ -346,9 +346,9 @@ class StatisticsChart extends LitElement {
         const dataValues: Array<number | null> = [];
         statTypes.forEach((type) => {
           let val: number | null;
-          if (type === "sum") {
+          if (type === "sum" || type === "sum_rel") {
             if (initVal === null) {
-              initVal = val = stat.state || 0;
+              initVal = val = type === "sum_rel" ? 0 : stat.sum || 0;
```

Comment:
Should this be `stat.sum` or `stat.state`? We want the total right, that would be `stat.state`, as `stat.sum` could have been reset somewhere?

Reply:
It should be `stat.sum`.
The `stat.state` is just used as a scratch pad for sensor to remember its state in the previous period, and we should perhaps remove it from the WS API response.

Reply:
We discussed this a bit further, and the conclusion was we should in fact display `stat.state` as is.